src/test/unit/algorithms/SDRClassifierTest.cpp

```diff
@@ -85,15 +85,15 @@ TEST_F(SDRClassifierTest, Basic) {
         // The -1 key is used for the actual values
         ASSERT_EQ(false, foundMinus1) << "Already found key -1 in classifier result";
         foundMinus1 = true;
-        ASSERT_EQ(5, it->second->size())
+        ASSERT_EQ(5ul, it->second->size())
```

Comment:
I wonder why these were not picked up earlier.  Do I have a different setting in the gtest install maybe?  The errors travis is still finding in the test modules seem to be gtest related as well.

Anyway I have to leave this morning so if you want to see if you can find the problem help your self.  If not, I will take it on when I get back next week.  I will check my email at least once a day but will not have facilities to do any coding.

Reply:
https://travis-ci.org/htm-community/nupic.cpp/builds/471251740#L1042

It finds an error in included external header. These should be included with ‘-isystem ‘ instead of-I to avoid that kind of error

Reply:
So, how do we tell CMake to do that?